src/main/resources/types.graphqls

```diff
@@ -17,6 +17,7 @@ input EnvironmentNodeFilterInput {
 input TargetNodesFilterInput {
     id: Int
     name: String
+    names: [String]
```

Comment:
Change looks good! Could you also add this `names: [String]` filter to the other filter inputs? `EnvironmentNodeFilterInput`, `DescendantTargetsFilterInput`, `ActiveRecordingFilterInput`, and `ArchivedRecordingFilterInput`. I think the implementation will be basically just a copy-paste job across a few other files, and the tests should all be very similar as well.

Reply:
will sure do